src/Cli.cpp

```diff
@@ -50,6 +51,15 @@ int Cli::main(int argc, char **argv) {
         save_mesh(polygon_mesh);
     }
 
+    // Do checks if user wants to do slicing here...
+    // then launch the slicer like this:
+    /*
+    if (Slicer::enabled()) {
+        Slicer slicer;
+        slicer.launch(Cura, "mesh.stl");
+    }
+    */
+
```

Comment:
Är inte "utkommenterad" kod på så sätt utan jag tänkte be dig lägga in det i CLI:t och så la jag bara en liten kommentar för att visa hur det var tänkt att använda klassen.